cfy_cluster_setup/cluster_config.yaml

```diff
@@ -23,56 +23,65 @@ existing_vms:
     manager-1:
       private_ip: ''
       public_ip: ''
-      cert_path: ''
-      key_path: ''
+      hostname: ''   # As specified in the certificate (if specified). Optional
```

Comment:
What about making public_ip and hostname both default to private IP and putting that in the comments? Might make for a better UX for simpler cluster cases?

Reply:
Hmm... or maybe hostname shouldn't- but what do we actually use it for in cloudify-manager-install?

Reply:
The hostname is used in two places:
1. If supplied, we'll check if it's specified in the instance certificate's SAN as `DNS:<hostname>`. 
2. If supplied, it will be specified in the config.yaml under manager.hostname.

The user can leave it blank, so we might not want to default it to the private-ip. But I do agree we should default the public-ip to the private-ip.

Reply:
OK- we should probably check what we do with it when it's in the config.yaml at some point too, but that's tangential to this.